Libs/MRML/Core/vtkMRMLTransformableNode.cxx

```diff
@@ -141,6 +141,45 @@ void vtkMRMLTransformableNode::ProcessMRMLEvents ( vtkObject *caller,
     }
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLTransformableNode
+::OnNodeReferenceAdded(vtkMRMLNodeReference* reference)
+{
+  Superclass::OnNodeReferenceAdded(reference);
+  if (std::string(reference->GetReferenceRole()) == this->TransformNodeReferenceRole)
+    {
+    this->InvokeCustomModifiedEvent(
+      vtkMRMLTransformableNode::TransformModifiedEvent, reference->GetReferencedNode());
+    }
+
+  vtkMRMLTransformNode* transformNode =
+    vtkMRMLTransformNode::SafeDownCast(reference->GetReferencedNode());
+  if (transformNode)
+    {
+    transformNode->InvokeEvent(
+      vtkMRMLTransformNode::TransformReferenceModifiedEvent, this);
```

Comment:
That's a good point, yes, that would be great !
Isn't this something that should be tackled in another issue/PR though ? It seems like it would be better to do this as a separate branch.